AppCenterAnalytics/AppCenterAnalytics/Internals/MSAnalyticsInternal.h

```diff
@@ -14,10 +14,14 @@ NS_ASSUME_NONNULL_BEGIN
  * @param eventName  Event name.
  * @param properties The typed event properties.
  * @param transmissionTarget  The transmission target to associate to this event.
+ * @param flags      Optional flags. Events with MSFlagsPersistenceCritical will be considered as a higher priority than events with
+ * MSFlagsPersistenceNormal or MSFlagsDefault, will be removed at the latest when the storage is full and sent prior to lower priority
```

Comment:
will be removed at the latest ... when, something is missing between latest and when, maybe some wording can be improved here